rskj-core/src/test/java/co/rsk/peg/BridgeTest.java

```diff
@@ -640,10 +642,177 @@ public void getBtcBlockchainBestChainHeightOnlyAllowsLocalCalls_beforeRskip220()
         Assert.assertTrue(bridge.getBtcBlockchainBestChainHeightOnlyAllowsLocalCalls(new Object[0]));
     }
 
+    @Test
+    public void activeAndRetiringFederationOnly_activeFederationIsNotFromFederateMember_retiringFederationIsNull_throwsVMException() throws Exception {
```

Comment:
it seems this two tests have the same behavior just different inputs. I mean they create the same environment and perform the same operations. just expects different.

lets remove this code duplication and encapsulate them

Reply:
Done!